### PR TITLE
Swa 71 integrate microsoft

### PR DESCRIPTION
**Pull request by yuriyarabskyy**
Add microsoft transcriptions to the transcription server

### CODE REVIEW

File: server_transcriptor/external/microsoft.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    def recognize(self, audio_path):
+        try:
+            sound, sample_rate = librosa.load(audio_path, sr=RATE)
+            librosa.output.write_wav(TMP_WAV, sound, sample_rate)
+            return self.speech_recognize_once_from_file(TMP_WAV)
+        except KeyboardInterrupt:
+            pass
+
+
+def main(config):
```
Comment (WiktorJ) [inline server_transcriptor/external/microsoft.py]: Move this stuff to separate file, or completely remove

File: server_transcriptor/external/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+key: 8ae4c6464f7d42b19c690b7e3c10693e
```
Comment (WiktorJ) [inline server_transcriptor/external/secret.yaml]: should this be here?

Comment (yuriyarabskyy) [reply server_transcriptor/external/secret.yaml]: it shouldn't but it's just a trial account so nobody can really exploit it and it's easier right now for testing to just have it there

File: server_transcriptor/external/microsoft.py (hunk 1 above)
Comment (yuriyarabskyy) [reply server_transcriptor/external/microsoft.py]: yep, I'll move it to tests

**Pull request merged**